OWASP/common-requirement-enumeration
# Filtering experiment

testing instructions: 
terminal: cp cres/db.sqlite standards_cache.sqlite;  yarn start; make dev-run

in a browser: 
localhost:9001/standard/ASVS or localhost:9001/search/asvs
expand dropdowns and click on the "Filter" button
then click apply filter on the header bar
you should be able to see the links to the CRE-ids you have selected

the clear bytton should clear the url and re-show everything
